### ISSUE TITLE
Localization 

### ISSUE THREAD

**Issue by KovalevArtem** [Tyrrrz/LightBulb#183]
It is sad that the application is only in English - I propose to translate.
I am ready to help in the translation into Russian.
(For example, you can use Weblate, or any other service. It is completely free for open source projects.)

**Comment by Tyrrrz** [comment]
Duplicate of #45.

Yes, localization would be really nice. In the past I have already created an account for LightBulb on Crowdin, but it didn't get anywhere past that point. Currently there are still a few open questions:
- How should the localizable strings be stored? Is there a common/standard/popular format? I don't want to build a custom one if I could avoid it.
- How should the localization process be synchronized with development? What happens if new strings are added and there is no one to translate them?
- How should the localization be deployed? For example, releasing new features/bug fixes can't be delayed by waiting on localization, so it's likely they won't be deployed in tandem.
